### PR TITLE
Marcinzo/issuervalidatorv11

### PR DESCRIPTION
**Pull request by MZOLN**
# Add support for v11 to Aad Issuer Validator

## Description

Adds support for the new AAD stop gap v1.1 version

### CODE REVIEW

File: test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs [hunk 1]
```diff
@@ -595,11 +594,16 @@ public void Validate_FromB2CAuthority_WithTfpIssuer_ThrowsException()
...
-        public void Validate_WithAuthorityUsingConfigurationProvider(bool isV2Token, bool isV2Authority)
+        [InlineData(ProtocolVersion.V1, ProtocolVersion.V1)]
+        [InlineData(ProtocolVersion.V1, ProtocolVersion.V11)]
+        [InlineData(ProtocolVersion.V1, ProtocolVersion.V2)]
+        [InlineData(ProtocolVersion.V11, ProtocolVersion.V1)]
+        [InlineData(ProtocolVersion.V11, ProtocolVersion.V11)]
+        [InlineData(ProtocolVersion.V11, ProtocolVersion.V2)]
+        [InlineData(ProtocolVersion.V2, ProtocolVersion.V1)]
+        [InlineData(ProtocolVersion.V2, ProtocolVersion.V11)]
+        [InlineData(ProtocolVersion.V2, ProtocolVersion.V2)]
```
Comment (westin-m) [inline test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs:605]: Can this be CombinatorialData instead? Same question for the other two tests

**Review by FuPingFranco — APPROVED** [review]
LGTM

File: test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs (hunk 1 above)
Comment (MZOLN) [reply test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs:605]: I'm not aware of that. How would that look like? Would that be blocking a sign off?

Comment (westin-m) [reply test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs:605]: Not blocking. Here's the repo:
https://github.com/aarnott/Xunit.Combinatorial

**Review by westin-m — APPROVED** [review]

**Review by jennyf19 — CHANGES_REQUESTED** [review]
blocking the merge, as i would like to review, and @brentschmaltz and/or @jmprieur

File: test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs (hunk 1 above)
Comment (MZOLN) [reply test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs:605]: I'm not sure I would feel comfortable taking dependency on this.

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Review by jennyf19 — DISMISSED** [review]
waiting for further review.

**Review by keegan-caruso — APPROVED** [review]
LGTM

**Pull request merged**